tools/c7n_azure/c7n_azure/provisioning/deployment_unit.py

```diff
@@ -1,14 +1,12 @@
 import logging
-import six
 
 from abc import ABCMeta, abstractmethod
 
 from c7n.utils import local_session
 from c7n_azure.session import Session
 
 
-@six.add_metaclass(ABCMeta)
-class DeploymentUnit:
+class DeploymentUnit(metaclass=ABCMeta):

```

Comment:
does this need a metaclass?

Reply:
In this case, I think it's used.  For example, `class AutoScaleUnit` in `autoscale.py` inherits from `DeploymentUnit` and is forced to implement `_get` and `_provision`.  There are other classes that are similar.

Reply:
fair enough, thanks for checking